packages/calcite-components/src/components/tabs/tabs.tsx

```diff
@@ -133,6 +153,43 @@ export class Tabs {
    */
   @State() tabs: HTMLCalciteTabElement[] = [];
 
+  mutationObserver = createObserver("mutation", (mutationsList: MutationRecord[]) => {
+    console.log("mutationsList", mutationsList);
+    for (const mutation of mutationsList) {
+      const target = mutation.target as HTMLElement;
+      if (
+        target.nodeName === "CALCITE-TAB-NAV" ||
+        target.nodeName === "CALCITE-TAB-TITLE" ||
+        target.nodeName === "CALCITE-TAB"
+      ) {
+        this.updateItems();
+      }
+    }
+  });
+
+  private updateItems = (): void => {
+    const { position, scale } = this;
+
+    const nav = this.el.querySelector("calcite-tab-nav");
+    if (nav) {
+      nav.position = position;
+      nav.scale = scale;
+    }
+
+    Array.from(this.el.querySelectorAll("calcite-tab:not(.calcite-tab-nested)")).forEach(
+      (tab: HTMLCalciteTabElement) => {
+        tab.scale = scale;
+      }
+    );
+
+    Array.from(this.el.querySelectorAll("calcite-tab-title")).forEach(
```

Comment:
Can you update the selector so that it targets direct children of a `tabs`' direct `tab-nav` child?